build.json

```diff
@@ -1,7 +1,7 @@
 {
   "variable": {
     "PYTHON_VERSION": {
-      "default": "3.9.13"
+      "default": "3.11.3"
```

Comment:
```suggestion
      "default": "3.11.11"
```

Reply:
Unfortunately we can't easily update since those images have updated Jupiter stack with notebook v7, with which we are not compatible.

Reply:
Can you open a different PR for it so we can test it?

Reply:
sure